ros/src/computing/perception/detection/packages/road_wizard/launch/feat_proj.launch

```diff
@@ -8,6 +8,9 @@
   we need to regenerate filtered signal data filteredsignaldata.csv -->
   <arg name="use_filtered_signal_map" default="false"/>
   <arg name="filtered_signal_map_name" default="$(find demo_scripts)/config/filteredsignaldata.csv"/>
+  <!--
+  filteredsignaldata.csv is created manually. This file has the information of signals which are on the lane of our waypoints.
```

Comment:
What kind of information? Which format? I meant something like:

| signal_id  | pole_id | vector_id | point_id | lane_id |
| ------------- | ------------- | ------------- | ------------- | ------------- |
| uint | uint | uint | uint | uint |

Example contents of CSV :
```
1,1536,4445,1235,7856
3,756,5536,7852,843
```